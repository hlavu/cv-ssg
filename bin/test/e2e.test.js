const run = require("./run");

describe("end-to-end integration", () => {
  test("prints error and help message when no arguments given", async () => {
    const { stderr, stdout, exitCode } = await run();

    expect(exitCode).toBe(1);
    expect(stderr).toMatchSnapshot();
    expect(stdout).toEqual("");
  });
});
