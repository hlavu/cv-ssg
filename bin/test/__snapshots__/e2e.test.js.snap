// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`end-to-end integration prints error and help message when no arguments given 1`] = `
"cv-ssg.js <args> [options]

Options:
  --input, -i       input file or folder                                [string]
  --stylesheet, -s  use your custom stylesheet or <default> for default
                    stylesheet                            [string] [default: \\"\\"]
  --config, -c      input a config.json file containing other options' values
                                                          [string] [default: \\"\\"]
  -v, --version     Show version number                                [boolean]
  -h, --help        Show help                                          [boolean]"
`;
